homeassistant/components/meater/sensor.py

```diff
@@ -10,10 +21,108 @@
     CoordinatorEntity,
     DataUpdateCoordinator,
 )
+from homeassistant.util import dt as dt_util
 
 from .const import DOMAIN
 
 
+@dataclass
+class MeaterSensorEntityDescription(SensorEntityDescription):
+    """Describes meater sensor entity."""
+
+    available: Callable = lambda x: NotImplementedError
+    value: Callable = lambda x: NotImplementedError
```

Comment:
We could make this a little more specific to aid with typing?

```suggestion
    available: Callable[[MeaterProbe | None], bool] = lambda x: NotImplementedError
    value: Callable[[MeaterProbe], str | float | None] = lambda x: NotImplementedError
```

(not entirely sure on the latter, might need `int` too?)